goal/src/main/java/org/openapitools/vertxweb/server/api/ListPetsArguments.java

```diff
@@ -0,0 +1,12 @@
+package org.openapitools.vertxweb.server.api;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.ext.web.RoutingContext;
+
+@VertxGen
+public interface ListPetsArguments {
+    // always present (maybe derive from a base class?)
+    RoutingContext context();
```

Comment:
Probably this shouldn't be present, but access to the context data can be interesting as a way to read data created in the intermediate handlers

Maybe a:

```java
<T> T get(String key);
```

Which is just delegating to the context anyway.

Reply:
I wonder, do we really need to have the `Arguments` interface? Couldn't we just pass the arguments in the method?